lib/templates/deploy.batch.dnx.consoleapp.template

```diff
@@ -0,0 +1,21 @@
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+:: Deployment
+:: ----------
+
+echo Handling DNX Console Application deployment.
+
+:: 1. Set WebJob deployment target
+IF NOT DEFINED WEBJOB_DEPLOYMENT_TARGET (
+  SET WEBJOB_DEPLOYMENT_TARGET=%DEPLOYMENT_TARGET%\App_Data\jobs\continuous\deployedJob
+)
+
+:: 2. Create WebJob directory
+IF EXIST "WEBJOB_DEPLOYMENT_TARGET" (
+  CALL :ExecuteCmd mkdir "%WEBJOB_DEPLOYMENT_TARGET%"
+)
+
+:: 3. KuduSync
+CALL :ExecuteCmd "%KUDU_SYNC_CMD%" -v 50 -f "{WEBJOB_PROJECT_PATH}" -t "%WEBJOB_DEPLOYMENT_TARGET%" -n "%NEXT_MANIFEST_PATH%" -p "%PREVIOUS_MANIFEST_PATH%" -i ".git;.hg;.deployment;deploy.cmd"
```

Comment:
I just checked and it doesn't have that issue. e.g. deploying https://github.com/KuduApps/ConsoleWorker, I ended with this manifest:

```
hostingstart.html
app_data
app_data\jobs
app_data\jobs\continuous
app_data\jobs\continuous\deployedJob
app_data\jobs\continuous\deployedJob\ConsoleWorker.exe
app_data\jobs\continuous\deployedJob\ConsoleWorker.exe.config
app_data\jobs\continuous\deployedJob\ConsoleWorker.pdb
app_data\jobs\continuous\deployedJob\settings.job
```